app/src/styles.sass

```diff
@@ -42,8 +43,19 @@ input
     &[disabled], &[disabled]:hover
       border-color: tame($maincolor)
       background-color: tame($userbuttonbgcolor)
-      color: tame($useboxfgcolor)
+      color: tame($useboxfgcolor) 
 
+@media screen and (max-width: $breakpoint-tablet)
+  .dialog-box .formactions
+    padding: 0
+    display: flex
+    justify-content: center
```

Comment:
This line is useless, because there is no external constraint on the height of the element.